.github/workflows/build.yaml

```
@@ -103,21 +97,18 @@ jobs:
         strategy:
             matrix:
                 php-version:
-                    - "7.4"
-                    - "8.0"
                     - "8.1"
+                    - "8.2"
+                    - "8.3"
+                    - "8.4"
 
                 dependencies:
                     - "highest"
 
                 symfony:
-                    - "^4.4"
-                    - "^5.4"
-                    - "^6.0"
-
-                exclude:
-                    -   php-version: "7.4"
-                        symfony: "^6.0"
+                    - "^6.4"
+                    - "^7.1"
+                    - "^7.2"
```

```suggestion
                    - "~6.4.0"
                    - "~7.1.0"
                    - "~7.2.0"
```

Should look like that